src/olympia/amo/admin.py

```diff
@@ -2,14 +2,40 @@
 import operator
 
 from django.contrib import admin
+from django.contrib.admin.views.main import ChangeList, ChangeListSearchForm, SEARCH_VAR
 from django.core.exceptions import FieldDoesNotExist
 from django.db import models
 from django.db.models.constants import LOOKUP_SEP
 
 from .models import FakeEmail
 
 
+class CommaSearchInAdminChangeListSearchForm(ChangeListSearchForm):
+    def clean(self):
+        self.cleaned_data = super().clean()
+        search_term = self.cleaned_data[SEARCH_VAR]
+        if ',' in search_term:
+            self.cleaned_data[SEARCH_VAR] = ','.join(
+                [
+                    term.strip()
+                    for term in search_term.strip().split(',')
+                    if term.strip()
+                ]
+            )
```

Comment:
```suggestion
        self.cleaned_data[SEARCH_VAR] = ','.join(
            term.strip() for term in self.cleaned_data[SEARCH_VAR].split(',')
        )
```
?

I couldn't see what all the extra `strip()`s were achieving.  (And, sure, always doing it on the string even if no `,` is wasteful but simple code tho.)

Reply:
I want to remove whitespace around the values but I also don't want to keep empty values: passing `a, ,b` should result in `a,b`. Similarly I want to ignore trailing space: `a,b,` should result in `a,b`, though I guess just doing the `if term.strip()` is enough for that and I can drop the `.strip()` on `search_term`.